qmlbind/include/qmlbind/engine.h

```diff
@@ -121,12 +121,14 @@ QMLBIND_API qmlbind_value *qmlbind_engine_new_array(qmlbind_engine *self, int le
  *
  * If `object` is a null pointer, this function returns a null value.
  *
+ * After creating a wrapper, metaclass changes such as method addition will not be reflected in the created wrapper.
+ *
```

Comment:
Let's add a note here about the required lifetime of the passed metaclass, too. Again, a client could destoy the metaclass directly after creating the wrapper, if he won't create any new wrappers anymore.

The client will probably implement a function that generates metaclasses for him, so he could plan to destroy each metaclass directly after use and create a new one later on, even if it's identical (has the same className, methods, properties etc.) I'm not sure whether this will lead to internal blow-up or strange side effects, but maybe we should still discourage this, and require that all metaclasses passed to `_new_wrapper` and `_metaclass_register` that have the same name have to be the identical metaclass pointer.

I find it really hard to express this in an understandable way, though. :sweat: